scripts/zip_quickstart_test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
This is a streamlined version of https://github.com/firebase/quickstart-ios/blob/master/scripts/framework_test.sh now modifiable in this repo.